test/e2e/overridepolicy_test.go

```diff
@@ -425,6 +429,198 @@ var _ = ginkgo.Describe("[OverridePolicy] apply overriders testing", func() {
 				})
 		})
 	})
+
+	ginkgo.Context("[FieldOverrider] apply field overrider testing to update JSON values in ConfigMap", func() {
+		var configMapNamespace, configMapName string
+		var configMap *corev1.ConfigMap
+
+		ginkgo.BeforeEach(func() {
+			configMapNamespace = testNamespace
+			configMapName = configMapNamePrefix + rand.String(RandomStrLength)
+			propagationPolicyNamespace = testNamespace
+			propagationPolicyName = configMapName
+			overridePolicyNamespace = testNamespace
+			overridePolicyName = configMapName
+
+			configMapData := map[string]string{
+				"deploy.json": fmt.Sprintf(`{
+				"apiVersion": "apps/v1",
+				"kind": "Deployment",
+				"metadata": {
+					"name": "nginx-deploy",
+					"namespace": "%s"
+				},
+				"spec": {
+					"replicas": 3,
+					"selector": {
+						"matchLabels": {
+							"app": "nginx"
+						}
+					},
+					"template": {
+						"metadata": {
+							"labels": {
+								"app": "nginx"
+							}
+						},
+						"spec": {
+							"containers": [
+								{
+									"name": "nginx",
+									"image": "nginx:1.19.0"
+								}
+							]
+						}
+					}
+				}
+			}`, configMapNamespace),
+			}
+
+			configMap = helper.NewConfigMap(configMapNamespace, configMapName, configMapData)
+			propagationPolicy = helper.NewPropagationPolicy(propagationPolicyNamespace, propagationPolicyName, []policyv1alpha1.ResourceSelector{
+				{
+					APIVersion: configMap.APIVersion,
+					Kind:       configMap.Kind,
+					Name:       configMap.Name,
+				},
+			}, policyv1alpha1.Placement{
+				ClusterAffinity: &policyv1alpha1.ClusterAffinity{
+					ClusterNames: framework.ClusterNames(),
+				},
+			})
+
+			overridePolicy = helper.NewOverridePolicy(overridePolicyNamespace, overridePolicyName, []policyv1alpha1.ResourceSelector{
+				{
+					APIVersion: configMap.APIVersion,
+					Kind:       configMap.Kind,
+					Name:       configMap.Name,
+				},
+			}, policyv1alpha1.ClusterAffinity{
+				ClusterNames: framework.ClusterNames(),
+			}, policyv1alpha1.Overriders{
+				FieldOverrider: []policyv1alpha1.FieldOverrider{
+					{
+						FieldPath: "/data/deploy.json",
+						JSON: []policyv1alpha1.JSONPatchOperation{
+							{
+								SubPath:  "/spec/replicas",
+								Operator: policyv1alpha1.OverriderOpReplace,
+								Value:    apiextensionsv1.JSON{Raw: []byte(`5`)},
```

Comment:
This test only covers `replace` operation, shall we cover `add` and `remove` as well?